src/lint/track_config.nim

```diff
@@ -293,7 +293,16 @@ type
 
   Concepts* = seq[Concept]
 
+  FilePatterns* = object
+    solution*: seq[string]
+    test*: seq[string]
+    example*: seq[string]
+    exemplar*: seq[string]
+    editor*: seq[string]
+
   TrackConfig* = object
+    slug*: string
+    files*: FilePatterns
```

Comment:
I've considered making this an `Option[FilePatterns]` value, but the default handling (making the patterns be empty sequences) works well in this case. Let me know what you think.